pkg/cri/sbserver/sandbox_stop.go

```diff
@@ -141,7 +142,8 @@ func (c *criService) teardownPodNetwork(ctx context.Context, sandbox sandboxstor
 	err = netPlugin.Remove(ctx, id, path, opts...)
 	networkPluginOperations.WithValues(networkTearDownOp).Inc()
 	networkPluginOperationsLatency.WithValues(networkTearDownOp).UpdateSince(netStart)
-	if err != nil {
+	// TODO(rongfu.leng) this is use netPlugin.Check() will be better, but cniVersion < 0.4.0 check is not supported, current loopback use cniVersion is 0.3.1
+	if err != nil && !strings.Contains(err.Error(), "failed to find plugin") {
```

Comment:
If the plugin is deleted after the pod is started, when the pod is deleted and the error return is skipped here, the network resources created by the plugin before will not be cleaned up and the user will not know about it, right?

Reply:
yes, this happens.

Reply:
This effectively allows the tear down of the plugin however with a significant limitation of not actually doing any sort of network clean that the CNI DEL would do? This could impact several areas such as IPAM. While the network namespace deletion would delete the veth, it could leave more resources around. How does the cni plugins get removed here? We should focus on that